packages/platform-sdk/src/contracts/coins/wallet-discovery.ts

```diff
@@ -0,0 +1,9 @@
+export interface AddressDiscoveryDataTransferObject {
```

Comment:
This is one of the rare exceptions where I would use the `DTO` acronym instead of `DataTransferObject` across the codebase